src/gobexport/objectstore.py

```diff
@@ -0,0 +1,52 @@
+"""Objectstore
+
+Encapsulates a file read from Objectstore into an iterator
+
+"""
+from gobcore.database.connector import connect_to_objectstore
+from gobcore.database.reader import query_objectstore
+
+from gobexport.config import get_objectstore_config
+
+
+class Objectstore:
```

Comment:
Rename to ObjectstoreFile or something? 'Objectstore' is a bit generic and misleading when reading the code.